worlds/stardew_valley/options.py

```diff
@@ -344,6 +390,115 @@ class Museumsanity(Choice):
+    Split Eradication Goals: The Monster Eradication Goals are split by monsters, each monster has one check
+    """
+    internal_name = "monstersanity"
+    display_name = "Monstersanity"
+    default = 1
+    option_none = 0
+    option_one_per_category = 1
+    option_one_per_monster = 2
+    option_goals = 3
+    option_short_goals = 4
+    option_very_short_goals = 5
+    option_progressive_goals = 6
+    option_split_goals = 7
+
+
+class Shipsanity(Choice):
+    """Locations for shipping items?
+    None: There are no checks for shipping items
+    Crops: Every crop and forageable being shipped is a check
+    Fish: Every fish being shipped is a check except legendaries
+    Full Shipment: Every item in the Collections page is a check
+    Full Shipment With Fish: Every item in the Collections page and every fish is a check
+    Everything: Every item in the game that can be shipped is a check
+    """
+    internal_name = "shipsanity"
+    display_name = "Shipsanity"
+    default = 0
+    option_none = 0
+    option_crops = 1
+    # option_quality_crops = 2
+    option_fish = 3
+    # option_quality_fish = 4
+    option_full_shipment = 5
+    # option_quality_full_shipment = 6
+    option_full_shipment_with_fish = 7
+    # option_quality_full_shipment_with_fish = 8
+    option_everything = 9
+    # option_quality_everything = 10
+
+
+class Cooksanity(Choice):
+    """Locations for cooking food?
+    None: There are no checks for cooking
+    Queen of Sauce: Every Queen of Sauce Recipe can be cooked for a check
+    All: Every cooking recipe can be cooked for a check
+    """
+    internal_name = "cooksanity"
+    display_name = "Cooksanity"
+    default = 0
+    option_none = 0
+    option_queen_of_sauce = 1
+    option_all = 2
+
+
+class Chefsanity(NamedRange):
+    """Locations for leaning cooking recipes?
+    Vanilla: All cooking recipes are learned normally
+    Queen of Sauce: Every Queen of sauce episode is a check, all queen of sauce recipes are items
+    Purchases: Every purchasable recipe is a check
+    Friendship: Recipes obtained from friendship are checks
+    Skills: Recipes obtained from skills are checks
+    All: Learning every cooking recipe is a check
+    """
+    internal_name = "chefsanity"
+    display_name = "Chefsanity"
+    default = 0
+    range_start = 0
+    range_end = 15
+
+    option_none = 0b0000  # 0
+    option_queen_of_sauce = 0b0001  # 1
+    option_purchases = 0b0010  # 2
+    option_qos_and_purchases = 0b0011  # 3
+    option_skills = 0b0100  # 4
+    option_friendship = 0b1000  # 8
+    option_all = 0b1111  # 15
+
+    special_range_names = {
+        "none": 0b0000,  # 0
+        "queen_of_sauce": 0b0001,  # 1
+        "purchases": 0b0010,  # 2
+        "qos_and_purchases": 0b0011,  # 3
+        "skills": 0b0100,  # 4
+        "friendship": 0b1000,  # 8
+        "all": 0b1111,  # 15
+    }
```

Comment:
Because players can use the flags structure to create their own combination, they don't need to choose one of the combinations I'm providing, but I'm providing some for simplicity's sake